.github/workflows/nodejs.yml

```diff
@@ -26,9 +26,11 @@ jobs:
         node-version: 
           # action based on https://github.com/actions/node-versions/releases
           # see also: https://nodejs.org/en/about/releases/
-          - "16"  # active LTS
+          - "17"      # current
```

Comment:
Seeing this just kinda reminded me that odd: 6 months, and even, 30 months. Can you start testing 18 yet or is it still pie in the sky?